test/e2e/aws_test.go

```diff
@@ -138,8 +123,87 @@ var _ = Describe("functional tests", func() {
 
 		})
 	})
+
+	Describe("multiple workload clusters", func() {
+		Context("in different namespaces", func() {
+			var ns1, clName1, ns2, clName2 string
+			It("should create first cluster", func() {
+				ns1 = namespace
+				clName1 = clusterName
+				By("Creating first cluster with single control plane")
+				makeSingleControlPlaneCluster(namespace, clusterName, awsClusterName, cpAWSMachinePrefix, cpBootstrapConfigPrefix, cpMachinePrefix, instanceType, testTmpDir)
+
+				By("Creating the MachineDeployment")
+				createMachineDeployment(namespace, clusterName, machineDeploymentName, awsMachineTemplateName, mdBootstrapConfig, instanceType, initialReplicas)
+			})
+			It("should create second cluster in a different namespace", func() {
```

Comment:
Making a new spec keeps the code clean but on the down side, increases the num of specs which may not be acceptable. Will be happy to hear any suggestions.

Reply:
I think the additional specs are fine here, it will give us room to optimize runtime by running them in parallel in the future.